chainer/functions/activation/tanh.py

```diff
@@ -57,13 +57,25 @@ def backward_gpu(self, x, gy):
 def tanh(x, use_cudnn=True):
     """Elementwise hyperbolic tangent function.
 
+     .. math:: f(x)=\\tanh(x).
+
     Args:
-        x (~chainer.Variable): Input variable.
+        x (:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`):
+            Input variable. A :math:`(s_1, s_2, ..., s_n)`-shaped float array.
         use_cudnn (bool): If ``True`` and cuDNN is enabled, then this function
             uses cuDNN as the core implementation.
 
     Returns:
-        ~chainer.Variable: Output variable.
+        ~chainer.Variable: Output variable. A
+        :math:`(s_1, s_2, ..., s_n)`-shaped float array.
+
+    .. admonition:: Example
+
+        >>> x = np.random.uniform(0, 1, (3, 4)).astype('f')
```

Comment:
Do we need this? `It is an element-wise operation, so the shape of the output array is the same as the input array.`
Because, we have already shown that the shapes of Args and Returns are same, (L65 and L71). But if it is better to describe it here as well, no problem to add this sentence in the example!